en/releases/main.md

```diff
@@ -69,7 +69,14 @@ Please continue reading for [upgrade instructions](#upgrade-guide).
 
 ### Fixed-wing
 
-- TBD
+- [Simplified airspeed sensor configuration](../config_vtol/vtol_without_airspeed_sensor.md): 
```

Comment:
Note, that if this is going into v1.15 we'll have to move this text to `releases/v1.15.md`